config/sockets.js

```diff
@@ -2,63 +2,30 @@
-   * Sails server with the most available resources. However that means that  *
-   * all room/pubsub/socket processing and shared memory has to be offloaded  *
-   * to a shared, remote messaging queue (usually Redis)                      *
-   *                                                                          *
-   * Luckily, Socket.io (and consequently Sails.js) apps support Redis for    *
-   * sockets by default. To enable a remote redis pubsub server, uncomment    *
-   * the config below.                                                        *
-   *                                                                          *
-   * Worth mentioning is that, if `adapter` config is `redis`, but host/port  *
-   * is left unset, Sails will try to connect to redis running on localhost   *
-   * via port 6379                                                            *
-   *                                                                          *
-   ***************************************************************************/
-  // adapter: 'memory',
-  //
-  // -OR-
-  //
-  // adapter: 'redis',
-  // host: '127.0.0.1',
-  // port: 6379,
-  // db: 'sails',
-  // pass: '<redis auth password>',
-  /***************************************************************************
+   * `transports`                                                             *
    *                                                                          *
-   * Whether to expose a 'get /__getcookie' route with CORS support that sets *
-   * a cookie (this is used by the sails.io.js socket client to get access to *
-   * a 3rd party cookie and to enable sessions).                              *
+   * The protocols or "transports" that socket clients are permitted to       *
+   * use when connecting and communicating with this Sails application.       *
    *                                                                          *
-   * Warning: Currently in this scenario, CORS settings apply to interpreted  *
-   * requests sent via a socket.io connection that used this cookie to        *
-   * connect, even for non-browser clients! (e.g. iOS apps, toasters, node.js *
-   * unit tests)                                                              *
+   * > Never change this here without also configuring `io.sails.transports`  *
+   * > in your client-side code.  If the client and the server are not using  *
+   * > the same array of transports, sockets will not work properly.          *
+   * >                                                                        *
+   * > For more info, see:                                                    *
+   * > https://sailsjs.com/docs/reference/web-sockets/socket-client           *
    *                                                                          *
    ***************************************************************************/
-  // grant3rdPartyCookie: true,
+  // transports: [ 'websocket' ],
```

Comment:
Interesting to see this. I wonder if using the websocket transport protocol offers console visibility into traffic